shared/common-adapters/avatar.js.flow

```diff
@@ -1,27 +1,30 @@
 // @flow
 import {Component} from 'react'
 
-export type AvatarSize = 176 | 112 | 80 | 64 | 56 | 48 | 40 | 32 | 24 | 16
+export type URLMap = {
+  '200': string,
+  '360': string,
+  '40': string,
+}
+
+export type AvatarSize = 176 | 112 | 80 | 64 | 48 | 40 | 32 | 24 | 16
 export type UserPictureSize = 360 | 200 | 40
-export type AvatarLookupCallback = (username: string, url: ?string) => void
-export type AvatarLookup = (username: string) => ?string
+export type AvatarLookupCallback = (username: string, urlMap: ?URLMap) => void
+export type AvatarLookup = (username: string) => ?URLMap
 export type AvatarLoad = (username: string, callback: AvatarLookupCallback) => void
 
 export type Props = {
-  size: AvatarSize,
-  style?: ?Object,
-  onClick?: ?(() => void),
-  onPress?: void,
-  // Use a solid color instead of a noAvatar image initially
-  loadingColor?: string,
   borderColor?: string,
-  forceLoading?: boolean,
+  children?: any,
   following?: ?boolean,
   followsYou?: ?boolean,
-  backgroundColor?: string,
-  opacity?: number,
-  children?: any,
+  loadingColor?: string,
   onAvatarLoaded?: () => void,
+  onClick?: ?(() => void),
+  opacity?: number,
+  size: AvatarSize,
+  style?: ?Object,
+  // TODO flow type to allow one or the other, not both url and username etc
```

Comment:
no, causes all sorts of problems. seems like the tagged variant stuff doesn't work well if you don't have a tag (but that would require us putting in a bunch of new props all over the place). (but flow is mysterious so maybe there's a way?) i discussed this w/ someone and i think we landed on it being impossible right now (marco maybe)